src/main/java/uk/gov/mca/beacons/service/accounts/AccountsController.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.mca.beacons.service.accounts;
+
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.mca.beacons.service.dto.AccountHolderIdDTO;
+import uk.gov.mca.beacons.service.exceptions.ResourceNotFoundException;
+import uk.gov.mca.beacons.service.model.AccountHolder;
+
+@RestController
+@RequestMapping("/account-holder")
+@Tag(name = "Account Holder")
+public class AccountsController {
```

Comment:
I will let you decide whether you think it is worth splitting the controller up into separate ones or not.

If we had multiple endpoints for account holders we could have them all exposed under this controller but have separate services/use cases for performing business logic.